net.c

```diff
@@ -559,8 +559,13 @@ deliver_to_host(struct qitem *it, struct mx_hostentry *host)
 	READ_REMOTE_CHECK("MAIL FROM", 2);
 
 	/* XXX send ESMTP ORCPT */
-	send_remote_command(fd, "RCPT TO:<%s>", it->addr);
-	READ_REMOTE_CHECK("RCPT TO", 2);
+	addrtmp = strdup(it->addr);
```

Comment:
probably we should check for an allocation error here and bail out with a recoverable error

Reply:
Good point, added a NULL check